graphbolt/build.bat

```diff
@@ -0,0 +1,31 @@
+REM Helper script to build Graphbolt libraries for PyTorch
+@ECHO OFF
+SETLOCAL EnableDelayedExpansion
+
+MD "%BINDIR%\graphbolt"
+DEL /S /Q build
+MD build
+PUSHD build
+
+IF x%1x == xx GOTO single
+
+FOR %%X IN (%*) DO (
+	DEL /S /Q *
```

Comment:
the indent here looks a bit large. it's 8 spaces?

Reply:
fixed